src/Models/ShallowWaterModels/example_zeroD.jl

```diff
@@ -0,0 +1,34 @@
+# # One dimensional ShallowWater example
```

Comment:
This example probably shouldn't be in the source; it should be in the `examples` (if we want it there).

Examples are fairly "expensive" because they need to be tested and maintained, so I think we should start with just one example for the shallow water equations (instead, we should tests to make sure everything works).

Reply:
Agreed.